mail/urlscan/Portfile

```diff
@@ -17,13 +17,16 @@ long_description    ${name} lets you easily browse URLs contained in an email\
 
 github.tarball_from archive
 
-python.default_version 39
+python.default_version 312
+python.versions 39 310 311 312
```

Comment:
this is only useful for ports that are called "py-<portname>" where it will automatically add subports. That is not applicable here, so please remove.

Reply:
Thanks. I have a question about this. `urlscan` can work with 3.9 and upwards. What is the best practice for macports for something like that? I hate to force people to install another version of Python unnecessarily if they prefer whatever they already have. Is there any way to mark the possible Python versions without being a `py-` package? Or are the subport-type features like that only available with those?

Reply:
I assume there isn't, based on the Guide's language. In that case: what does macports suggest when choosing a default version? The latest?

Reply:
It should use MacPort's default Python version whenever possible. That is to minimize the number of Python versions for the users to install. Currently, the default is 3.12 ; it will update to the newest version on January 1st each year.